Makefile

```
@@ -437,7 +437,7 @@ android-clean: ##@prepare Clean Gradle state
 android-ports: export FLOWSTORM_PORT := 7722
 android-ports: export TARGET := android-sdk
```

You need a default, and you need to use lazy assignment(`?=`) to make it overridable.
```suggestion
android-ports: export RCT_METRO_PORT ?= 8081
android-ports: export FLOWSTORM_PORT ?= 7722
android-ports: export TARGET := android-sdk
```